assets/sass/components/idea-hub/_googlesitekit-idea-hub-dashboard-cta.scss

```diff
@@ -19,73 +19,45 @@
 .googlesitekit-plugin {
 
 	.googlesitekit-idea-hub__dashboard-cta {
-		background-color: $c-white;
+		display: flex;
+		flex-direction: column;
 		height: 100%;
-		position: relative;
+		justify-content: space-between;
 
-		@media (min-width: $bp-tablet) {
-			display: flex;
-			flex-direction: column;
-			justify-content: center;
+		.googlesitekit-heading-4,
+		p {
+			margin-top: 0;
 		}
 
-		@media (min-width: $bp-xlarge) {
-			align-items: center;
-			flex-direction: row;
-			justify-content: flex-start;
-			padding: $grid-gap-desktop;
-		}
-	}
-
-	.googlesitekit-idea-hub__dashboard-cta__content {
-		display: inline-block;
-
-		@media (min-width: $bp-tablet) {
-			padding-left: $grid-gap-desktop;
+		.googlesitekit-heading-4 {
+			color: $c-shark;
 		}
 
-		h5 {
-			font-family: $f-secondary;
-			font-size: 22px;
-			font-weight: $fw-secondary-normal;
-			margin: 0;
-			text-align: left;
+		.googlesitekit-badge {
+			margin-left: $grid-gap-desktop / 2;
 		}
 	}
 
-	.googlesitekit-idea-hub__dashboard-cta__icon {
-		display: inline-block;
-		padding-bottom: $grid-gap-desktop;
+	.googlesitekit-idea-hub__dashboard-cta-body {
+		overflow: hidden;
+		text-align: center;
 
-		@media (min-width: $bp-xlarge) {
-			padding-bottom: 0;
+		svg {
+			margin-bottom: -12px;
```

Comment:
@aaemnnosttv No, it's not related to `grid-gap-desktop`. We need to manually move the SVG by those pixels to make sure it's completely flushed to the bottom of the container since the SVG contains some box shadow which we don't want to be seen.